code/modules/mob/living/simple_animal/friendly/nian_caterpillar.dm

```diff
@@ -0,0 +1,157 @@
+#define COCOON_WEAVE_DELAY 5 SECONDS
+#define COCOON_EMERGE_DELAY 15 SECONDS
+#define FLYSWATTER_DAMAGE_MULTIPLIER 14
```

Comment:
You have this defined/undefined in multiple files, can you move it somewhere in _defines so there's one source of truth?

Reply:
Implemented into mob_defines.dm and given auto doc comment.